scripts/lc_marc_update.py

```diff
@@ -22,6 +20,9 @@
 c = config.runtime_config['lc_marc_update']
 base_url = 'http://openlibrary.org'
 import_api_url = base_url + '/api/import'
+internal_error = '<Code>InternalError</Code>'
+no_bucket_error = '<Code>NoSuchBucket</Code>'
```

Comment:
Where did this content come from? How do we know it's correct?

Reply:
The technique it to grep the repo for the word in question and see where and how it is used.  These literals were used in two or three other files.